docs/tutorials/projects/send-security-photo.md

```diff
@@ -1,303 +1,303 @@
+
+If you are starting with this tutorial, follow the [steps here](/tutorials/projects/light-up/#write-python-code-to-control-your-object-detection-robot) to create the main script file, connect the code to the robot, and select the model and label path.
+Ignore the step about the Kasa smart plug host address.
+
+Instead of using this person detector to activate a smart plug, you will send yourself a text message.
+
+Make a copy of the [<file>lightupbot.py</file>](https://github.com/viam-labs/devrel-demos/blob/main/Light%20up%20bot/lightupbot.py) file in your project directory and save it as <file>chocolate_security.py</file>.
+You will use the same robot connection code and Vision Service configuration code but edit some other parts of the file.
+
+Delete the `from kasa import Discover, SmartPlug` line and replace it with the following to import the Yagmail Python library:
+
+```python
+import yagmail
+```
+
+Now you need to rewrite the if/else function.
+If a person is detected, your robot will print `sending a message`, take a photo, and save it to your computer as <file>foundyou.png</file> (or whatever name you want).
+
+Then you will create a `yagmail.SMTP` instance to initialize the server connection.
+
+Refer to the code below and the [yagmail instructions](#how-to-use-yagmail) to edit your <file>chocolate_security.py</file> file as necessary.
+
+{{% expand "Click to show the full example code." %}}
+
+```python {class="line-numbers linkable-line-numbers" data-line="6"}
+import asyncio
+import os
+
+from viam.robot.client import RobotClient
+from viam.rpc.dial import Credentials, DialOptions
+from viam.components.camera import Camera
+from viam.components.types import CameraMimeType
+from viam.services.vision import VisionClient, VisModelConfig, VisModelType, Detection
+import yagmail
+
+
+# These must be set. You can get them from your robot's 'CODE SAMPLE' tab
+robot_secret = os.getenv('ROBOT_SECRET') or ''
+robot_address = os.getenv('ROBOT_ADDRESS') or ''
+
+async def connect():
+    creds = Credentials(
+        type='robot-location-secret',
+        payload=robot_secret)
+    opts = RobotClient.Options(
+        refresh_interval=0,
+        dial_options=DialOptions(credentials=creds)
+    )
+    return await RobotClient.at_address(robot_address, opts)
+
+async def main():
+    robot = await connect()
+
+    print('Resources:')
+    print(robot.resource_names)
+    # This string should match your camera component name in your robot config on the Viam app
+    camera = Camera.from_robot(robot, "my-camera")
+    image = await camera.get_image()
+
+    # Get and set up the Vision Service
+    vision = VisionClient.from_robot(robot)
+    params = {"model_path": "./effdet0.tflite", "label_path": "./labels.txt", "num_threads": 1}
+    personDet = VisModelConfig(name="person_detector", type=VisModelType("tflite_detector"), parameters=params)
+    await vision.add_detector(personDet)
```

Comment:
This no longer works - and needs to be changed. Would you like to do that? Otherwise let's keep the alert at the top of this file

Reply:
yes, keeping alert